util/sshutil/transport_validation.go

```diff
@@ -0,0 +1,11 @@
+package sshutil
+
+import (
+	"regexp"
+)
+
+var gitSSHRegex = regexp.MustCompile("^[a-z0-9]+@[a-zA-Z0-9-.]+:.*$")
```

Comment:
dashes & underscores are also allowed in username

Reply:
good point, fixed.